docs/UserGuide.md

```diff
@@ -13,9 +13,20 @@
 </pic>
 </center>
 
-KonTActs is a **desktop app for managing your contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+KonTActs is a **desktop app designed for CS2030S teaching assistants (TAs) who works at the National University of Singapore, School of Computing.
 
-KonTActs streamlines contact management, helping you keep track of students, professors, and fellow TAs more effectively than traditional GUI apps.
+KonTActs streamlines contact management, helping you keep track of your students contact information more effectively compared to traditional GUI apps. 
```

Comment:
Maybe it would be better to say "contact, attendance and assignment information"